packages/user-ui/src/pages/profile/components/designed_table.rs

```diff
@@ -1,71 +1,67 @@
 use dioxus::prelude::*;
 use dioxus_translate::{translate, Language};
-use models::profile::{DesignProject, ProjectStatus, Role};
+use models::deliberation::{Deliberation, DeliberationStatus};
 use num_format::{Locale, ToFormattedString};
 
 use crate::{
-    components::icons::{
-        adopted::Adopted, in_progress::InProgress, waiting::Waiting, withdrawal::Withdrawal,
-    },
+    components::icons::{adopted::Adopted, in_progress::InProgress, waiting::Waiting},
     pages::profile::i18n::DesignedTableTranslate,
+    routes::Route,
     utils::time::format_prev_time,
 };
 
 #[component]
-pub fn DesignedTable(lang: Language, projects: Vec<DesignProject>) -> Element {
+pub fn DesignedTable(lang: Language, projects: Vec<Deliberation>, user_id: i64) -> Element {
     rsx! {
         div { class: "flex flex-col w-full justify-start items-start",
             TableHeader { lang }
 
             for project in projects {
-                TableRow { lang, project }
+                TableRow { lang, project, user_id }
             }
         }
     }
 }
 
 #[component]
-pub fn TableRow(lang: Language, project: DesignProject) -> Element {
+pub fn TableRow(lang: Language, project: Deliberation, user_id: i64) -> Element {
+    let nav = use_navigator();
     let tr: DesignedTableTranslate = translate(&lang);
 
-    let role = if !project.role.is_none() {
-        Role::to_type(&project.role.unwrap(), &lang)
-    } else {
-        "".to_string()
-    };
-
-    let number_of_participation = project
-        .num_of_participation
-        .to_formatted_string(&Locale::ko);
-    let prev_time = format_prev_time(project.created_at);
-    let status = ProjectStatus::to_type(&project.status, &lang);
+    let number_of_participation = project.response_count.to_formatted_string(&Locale::ko);
+    let prev_time = format_prev_time(project.updated_at);
+    let status = project.status();
 
-    let icon = match project.status {
-        ProjectStatus::Inprogress => rsx! {
-            InProgress {}
-        },
-        ProjectStatus::Withdrawal => rsx! {
-            Withdrawal {}
-        },
-        ProjectStatus::Waiting => rsx! {
+    let icon = match status {
+        DeliberationStatus::Ready => rsx! {
             Waiting {}
         },
-        ProjectStatus::Adopted => rsx! {
+        DeliberationStatus::InProgress => rsx! {
+            InProgress {}
+        },
+        DeliberationStatus::Finish => rsx! {
             Adopted {}
         },
     };
 
     rsx! {
-        div { class: "flex flex-row w-full min-h-[55px] bg-white border-b border-b-[#e6e6e6] font-normal text-[15px] text-[#222222]",
+        div {
+            class: "cursor-pointer flex flex-row w-full min-h-[55px] bg-white border-b border-b-[#e6e6e6] font-normal text-[15px] text-[#222222]",
```

Comment:
from now, apply our convention if you can,
- use predefined color in input.css
- define spacing variable, refer to other repos using tailwindcss4
- then use `min-h-55` instead of `[55px]` convention

Reply:
넵 작업하면서 수정해나가보도록 해보겠습니다!